### ISSUE TITLE
chroot for package building?

### ISSUE THREAD

**Issue by Schala** [msys2/MINGW-packages#6850]
Is there a way to make a chroot for package building? I was able to do so when I used Arch because of the devtools package, which has not been ported. I tried an explicit sysroot with pacman, but it was expecting some files already be in the chroot or something. I can't find any tutorials on this, and any help would be greatly appreciated.

**Comment by 1480c1** [comment]
Usually, it would just be downloading either the tarball or the sfx.exe file from https://github.com/msys2/msys2-installer/releases and then extracting msys2 into another folder and running msys2 from there. That's how I usually have 3+ concurrent installs running with different roots.


I mainly work as long as the binary paths relative to the bash you execute comes first (if I have separate msys2 installs in `C:\msys64` and `D:\msys64` and I have `C:\msys64\mingw64\bin;C:\msys64\usr\bin` in my PATH, and I want to launch `D:\msys64\usr\bin\bash.exe` I will need to prepend `D:\msys64\mingw64\bin;D:\msys64\usr\bin` to my PATH, this is automatically done with the mingw64 and mingw32 envs)

**Comment by Schala** [comment]
And the installer doesn't overwrite anything in the registry pertaining to other installs?

**Comment by jeremyd2019** [comment]
> And the installer doesn't overwrite anything in the registry pertaining to other installs?

No.  Especially the .sfx.exe is just a sfx 7zip archive.  But the 'installer' proper just extracts files and creates start menu shortcuts.

**Comment by 1480c1** [comment]
That's sort of why I mentioned `tarball` and `sfx.exe` since those are just tarballs that don't affect anything permanently, so you can just delete the folder if you want to "uninstall" it

**Comment by Schala** [comment]
Ah I see

**Comment by luzpaz** [comment]
Is this ticket still relevant ?
